libraries/anaconda3-python35/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ENV REQFILE=/home/algo/dependency.yml
```

Comment:
I feel like these should be unique file locations instead of stomping over each other

Reply:
yea that's why I was deleting the files at the end of the dockerfile ops, I was trying to keep things DRY but I think you're right, it's excessive. I'll add a suffix to the end like `dependency-python35.yml`